pyproject.toml

```
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "sunspecdemo"
+version = "0.0.0"
+description = "EPC SunSpec demonstration tool"
+authors = ["Alex Anker <alex.anker@epcpower.com>"]
+license = "MIT"
+readme = "README.rst"
+
+[tool.poetry-dynamic-versioning]
+enable = false
+vcs = "git"
+style = "pep440"
+
+[tool.poetry-dynamic-versioning.substitution]
+files = ["src/sunspecdemo/__init__.py"]
+
+[tool.poetry.dependencies]
+python = ">=3.7.9, <4.0"
```

```suggestion
python = ">=3.7.9, <4.0"
```
```suggestion
python = "^3.7.9"
```
These are equivalent statements.